src/k8s-frameworkcontroller/deploy/k8s-frameworkcontroller.yaml.template

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: k8s-frameworkcontroller-sts
+spec:
+  serviceName: k8s-frameworkcontroller
+  selector:
+    matchLabels:
+      app: k8s-frameworkcontroller
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: k8s-frameworkcontroller
+    spec:
+      containers:
+      - name: k8s-frameworkcontroller
+        image: frameworkcontroller/frameworkcontroller:v0.3.0
```

Comment:
Can we also limit the frameworkbarrier version in PAI runtime?
https://hub.docker.com/r/frameworkcontroller/frameworkbarrier/tags

Reply:
it already used v0.3.0